### PR TITLE
Case Info Details

### PR DESCRIPTION
**Pull request by EclipseIsDead**
This PR seeks to address Issue #8 by creating the info cards. Props will be passed from list to this card component and rendered onto DOM as child.

### CODE REVIEW

File: Frontend/src/CaseCard.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+              <Typography variant="body2">
+                Created {this.props.date} by {this.props.signature}
+              </Typography>
+            </CardContent>
+            <CardActions>
+              <Button size="small">Learn More</Button>
+            </CardActions>
+          </Card>
+        );
+    }
```
Comment (youssefsoli) [inline Frontend/src/CaseCard.js]: Add `export default CaseCard;` at the end.

File: Frontend/src/CaseCard.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import Card from '@mui/material/Card';
+import CardActions from '@mui/material/CardActions';
+import CardContent from '@mui/material/CardContent';
+import Button from '@mui/material/Button';
+import Typography from '@mui/material/Typography';
+
+class CaseCard extends React.Component{
```
Comment (youssefsoli) [inline Frontend/src/CaseCard.js]: Checking with @Sizbei and @dragonite20, will we be using classes or functions for our components?

File: Frontend/src/CaseCard.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+    constructor(props) {
+        super(props);
+      }
```
Comment (youssefsoli) [inline Frontend/src/CaseCard.js]: This constructor isn't required.

File: Frontend/src/CaseCard.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+               {this.props.clientName}
+              </Typography>
+              <Typography variant="h5" component="div">
+                Notes:
+              </Typography>
+              <Typography sx={{ mb: 1.5 }} color="text.secondary">
+                (details)
+              </Typography>
+              <Typography variant="body2">
+                Created {this.props.date} by {this.props.signature}
```
Comment (youssefsoli) [inline Frontend/src/CaseCard.js]: I am planning to pass in a full `user` object as a prop, so we can use `this.props.user` to access user attributes.

**Comment by youssefsoli** [comment]
Don't forget to put the component in the /src/components folder.

File: Frontend/src/CaseCard.js (hunk 4 above)
Comment (EclipseIsDead) [reply Frontend/src/CaseCard.js]: Kk, will work off of that then

File: Frontend/src/CaseCard.js (hunk 3 above)
Comment (EclipseIsDead) [reply Frontend/src/CaseCard.js]: Had it for if you implemented something that i might need it for, but seems like that's not the case so I'll remove it

**Comment by EclipseIsDead** [comment]
Screen Shot 2021-11-10 at 6 39 02 PM

**Pull request merged**